component/authenticator/pom.xml

```diff
@@ -54,6 +54,31 @@
             <version>13.0.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>2.9.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>converter-jackson</artifactId>
+            <version>2.9.0</version>
```

Comment:
move the versions as properties in parent pom

Reply:
resolved